cf-deployment.yml

```diff
@@ -431,8 +429,6 @@ instance_groups:
   instances: 2
   vm_type: small
   stemcell: default
-  update:
-    serial: true
```

Comment:
This `serial: true` only really ensures that the `diego-api` instance group happens before the `uaa` instance group. I performed a deploy where I move the position of `diego-api` after `uaa`, but left `serial: true` enabled. The deploy was successful, showing that `uaa` has no dependency on `diego-api` and it should be safe to deploy them in parallel.